src/pages/workspaces/workspace/Data.js

```diff
@@ -525,6 +537,11 @@ const EntitiesContent = ajaxCaller(class EntitiesContent extends Component {
         namespace, name,
         selectedEntities, selectedDataType: entityKey, runningSubmissionsCount
       }),
+      copyingEntities && h(ExportDataModal, {
+        onDismiss: () => this.setState({ copyingEntities: false }),
+        workspace,
+        selectedEntities, selectedDataType: entityKey, runningSubmissionsCount
```

Comment:
Also needs to change due to #930. Will need to be `selectedEntities: _.keys(selectedEntities)`. Whether now or later depends on which PR merges first. If this merges first, I'll take care of the change as part of #930.